addons/base_import/static/tests/import_action_tests.js

```diff
@@ -546,6 +546,42 @@ QUnit.module("Base Import Tests", (hooks) => {
         );
     });
 
+    QUnit.test("Import view: drag-and-drop file support", async function (assert) {
+        registerFakeHTTPService((route, params) => {
+            assert.strictEqual(route, "/base_import/set_file");
+            assert.strictEqual(
+                params.ufile[0].name,
+                "fake_file.csv",
+                "file is correctly uploaded to the server"
+            );
+        });
+        await createImportAction();
+
+        // Simulate a file drag and drop:
+        const dataTransfer = new DataTransfer();
+        dataTransfer.items.add(new File(["fake_file"], "fake_file.csv", {
+            type: "text/plain"
+        }));
+        const container = target.querySelector(".o_import_action");
+        container.dispatchEvent(new DragEvent("dragenter", {
+            bubbles: true,
+            dataTransfer
+        }));
+        await nextTick();
+        const dropzone = target.querySelector(".o-web-Dropzone");
+        dropzone.dispatchEvent(new DragEvent("drop", {
+            bubbles: true,
+            dataTransfer
+        }));
+        await nextTick();
```

Comment:
I will rather use the `dragenterFiles` and `dropFiles` helper functions to do that.